include/system/dev/vfs.h

```diff
@@ -28,6 +28,13 @@ struct fs_driver {
 	int (*isatty_r)(struct _reent*, void* const);
 	off_t (*lseek_r)(struct _reent*, void* const, off_t, int);
 	int (*ctl)(void* const, const uint32_t, void* const);
+    int (*mkdir_r)(struct _reent*, void* const);
+    int (*usd_link_r)(struct _reent*, void* const, char*, char*);
+    int (*usd_unlink_r)(struct _reent*, void* const, char*);
+    int (*usd_chdir_r)(struct _reent*, void* const, char*);
+    int (*usd_chmod_r)(struct _reent*, void* const, char*);
+    long (*usd_pathconf_r)(struct _reent*, void* const, char*, int);
```

Comment:
Why do these have names specific to drivers? The fs_driver struct is "instantiated" for each driver, so there's no need for that

Reply:
also in usd_driver.c (lines 101-108) you seem to have referred to them by what they should be, as opposed to what they actually are